ci-operator/step-registry/redhat-chaos/container-scenarios/etcd-hangup/redhat-chaos-container-scenarios-etcd-hangup-ref.yaml

```diff
@@ -50,6 +50,10 @@ ref:
       default: "0"
     - name: TELEMETRY_ARCHIVE_SIZE
       default: "1000000"
+    - name: TELEMETRY_LOGS_BACKUP
+      default: "True"
+    - name: TELEMTRY_CLI_PATH
+      default: "/usr/local/bin/oc"
```

Comment:
Should the default oc cli be in the path already?

Reply:
We copy the binary to /usr/local/bin - https://github.com/redhat-chaos/krkn/blob/main/containers/Dockerfile#L25, adding the path explicitly to make sure it finds it in cases where /usr/local/bin is not part of the PATH. Thoughts?